### ISSUE TITLE
Plus Server NDI stray markers

### ISSUE THREAD

**Issue by Zulex** [PlusToolkit/PlusLib#739]
Hi all,

tl;dr: No option to send positional data of stray markers in Plus server

Currently we are using the plusserver with success to stream tools to Unity, all on Windows. (Waiting for fork to be made of the unity project, will submit my improvements too.) 

The device we use is the NDI Polaris Spectra. I tried to add positional messages to the configuration file for the Polaris, but this seems to block any messages to be send (Couldn't find any documentation on changing this file to send stray markers). In the NDI tools itself, there is a flag to send stray markers too. I looked into the NDIcapi (which is used by plus server if read correctly), and this API seems to support stray markers. So I wondering if there is an way without building the server myself to send the positional data?

Please let me know if I'm missing any necessary information or overlooked anything.

**Comment by adamrankin** [comment]
Not too hard, but a little extra would have to be coded in to pass the data returned from ndicapi downstream. You'd have to make changes in https://github.com/PlusToolkit/PlusLib/blob/master/src/PlusDataCollection/NDICAPITracking/vtkPlusNDITracker.cxx to receive the stray markers from the api, then package them up as either metadata on the tracking message, or as separate messages themselves.

**Comment by Zulex** [comment]
Thanks for diving into this. Unfortunately we won't have time to make adjustments to the plus lib in our project (we are using just off the shelf plus now).  Opting for just placing a marker tool on our phantom, but hopefully this will be integrated in the future since it allows giving complete freedom in scenarios.
